Dockerfile.test

```diff
@@ -0,0 +1,38 @@
+# Usage:
```

Comment:
Why a new docker file and not reusing our standard image "docker.ci.diabeloop.eu/go-build:1.15" ?

Reply:
It's just for local testing, when you don't want to specifically start a mongodb for runnings theses tests.
It's not used in the Jenkins pipeline.
I found this docker script useful, so I propose it.
May not be included if you prefer.